test/models/test_retinaNet_decetron2.py

```diff
@@ -107,3 +105,17 @@ def get_detectron2_models_and_inputs_batch(model_yaml_path:  str = "COCO-Detecti
     tracer = Caffe2Tracer(cfg, model, sample_inputs) #this might raise w
     onnx_model = tracer.export_onnx() #this requires pip install onnx==1.8.1
     return onnx_model, model, sample_inputs
+
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize('change_ordering', [True, False])
```

Comment:
You can set change ordering to False as we use our own `change_ordering` and not the one of onnx2keras